dedibox-network/ipv6/how-to/configure-ipv6-networkd.mdx

```diff
@@ -68,7 +68,7 @@ When specifying the IP address of your Dedibox, you need to enter it in [CIDR no
     IPv6PrefixDelegation=dhcpv6
     ```
 
-7. Find the name of your public internet interface using the `ip link show` command and take a note of it. In this example, the name of the interface is `enp1s0f0`. Take a note of it, as it will be required in the next steps.
+7. Find the name of your public Internet interface using the `ip link show` command and take a note of it. In this example, the name of the interface is `enp1s0f0`. Take a note of it, as it will be required in the next steps.
```

Comment:
```suggestion
7. Find the name of your public Internet interface using the `ip link show` command. In this example, the name of the interface is `enp1s0f0`. Take a note of it, as it will be required in the next steps.
```